src/main/java/appeng/util/Platform.java

```diff
@@ -274,16 +275,7 @@ public static boolean checkPermissions(final PlayerEntity player, final Object a
     public static void spawnDrops(final World w, final BlockPos pos, final List<ItemStack> drops) {
         if (!w.isRemote()) {
             for (final ItemStack i : drops) {
-                if (!i.isEmpty()) {
-                    if (i.getCount() > 0) {
-                        final double offset_x = (getRandomInt() % 32 - 16) / 82;
-                        final double offset_y = (getRandomInt() % 32 - 16) / 82;
-                        final double offset_z = (getRandomInt() % 32 - 16) / 82;
-                        final ItemEntity ei = new ItemEntity(w, 0.5 + offset_x + pos.getX(),
-                                0.5 + offset_y + pos.getY(), 0.2 + offset_z + pos.getZ(), i.copy());
-                        w.addEntity(ei);
-                    }
-                }
+                InventoryHelper.spawnItemStack(w, pos.getX(), pos.getY(), pos.getZ(), i);
```

Comment:
This is something we cannot use as it will split the itemstacks into a shitton of additional entities. Worst case like a factor of 7 times more entities.

Reply:
Are we dropping that many item stacks?

Reply:
also wdym? how is this different from before?
i cant look at spawnItemStack vs. spawning ourselves

Reply:
We use it in a couple of places. Like destroying a crafting cpu and droping the inventory. Which is already bad.
But the helper will further split the stack into entities containing 10-30 items, so worst case 64 items turn into 7 entities and best case 3.

Reply:
Also this mutates the actually itemstack being passed to it. So no idea, if this isn't causing other side effects by not copying it.

Reply:
Now had a chance to look at this, and this is correct. And I would not have expected that based on the function name lol.

Reply:
Reverted, my bad.